### ISSUE TITLE
Multiple classes and the required prompt

### ISSUE THREAD

**Issue by sumitmishra209** [ShivamShrirao/diffusers#87]
I have a question, basically, I have multiple classes, so in the concept list I have added multiple 
{"instance_prompt", "class_prompt", "instance_data_dir", "class_data_dir"}. 

Further, while calling train_dreambooth.py, I removed argument --save_sample_prompt="photo of sks dog" \. 

If this is wrong ? Now for generating the image, will I need to call "instance_prompt" and  "class_prompt" together to get an image ?

Thanks.

**Comment by InB4DevOps** [comment]
> will I need to call "instance_prompt" and "class_prompt" together

yes

**Comment by sumitmishra209** [comment]
concepts_list = [
    {
        "instance_prompt":      "photo of sks dog",
        "class_prompt":         "photo of a dog",
        "instance_data_dir":    "/content/data/sks",
        "class_data_dir":       "/content/data/dog"
    },
    {
        "instance_prompt":      "photo of ukj person",
        "class_prompt":         "photo of a person",
        "instance_data_dir":    "/content/data/ukj",
        "class_data_dir":       "/content/data/person"
    }
]

**For the above concept list what should be --save_sample_prompt in the below code ?**

!accelerate launch train_dreambooth.py \
  --pretrained_model_name_or_path=$MODEL_NAME \
  --pretrained_vae_name_or_path="stabilityai/sd-vae-ft-mse" \
  --output_dir=$OUTPUT_DIR \
  --with_prior_preservation --prior_loss_weight=1.0 \
  --seed=1337 \
  --resolution=512 \
  --train_batch_size=1 \
  --train_text_encoder \
  --mixed_precision="fp16" \
  --use_8bit_adam \
  --gradient_accumulation_steps=1 \
  --learning_rate=1e-6 \
  --lr_scheduler="constant" \
  --lr_warmup_steps=0 \
  --num_class_images=50 \
  --sample_batch_size=4 \
  --max_train_steps=800 \
  --save_interval=10000 \
  --save_sample_prompt="photo of sks dog" \
  --concepts_list="concepts_list.json"

# Reduce the `--save_interval` to lower than `--max_train_steps` to save weights from intermediate steps.
# `--save_sample_prompt` can be same as `--instance_prompt` to generate intermediate samples (saved along with weights in samples directory).
